src/components/AllArticlesSortBy.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+import { Button } from "react-bootstrap";
+
+const AllArticlesSortBy = ({ setSearchParams }) => {
+  const [menuParams, setMenuParams] = useState({
+    sort_by: "",
+    order: "asc",
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
```

Comment:
This is a nice way of dealing with that interim bit of the values having been changed but not yet ready to be submitted